.importjs.js

```diff
@@ -1,27 +1,8 @@
-function isNodeFile(file) {
-  return file.indexOf('src/server') !== -1;
-}
 module.exports = {
-  environments: function({ pathToCurrentFile }) {
-    if (isNodeFile(pathToCurrentFile)) {
-      return ['node'];
-    }
-    return [];
-  },
+  environments: ['node'],
   excludes: [
     './lib/**',
+    './**/__tests__/**',
   ],
-  declarationKeyword: function({ pathToCurrentFile }) {
-    if (isNodeFile(pathToCurrentFile)) {
-      return 'const';
-    }
-    return 'import';
-  },
-  namedExports: {
-    'react': ['PropTypes'],
-    'js/src/Diff': [
-      'DiffController',
-    ]
-  },
-  importDevDependencies: true,
+  declarationKeyword: 'const',
```

Comment:
I've mostly just mirrored what the code looks like in the project. We can make a sweep to convert to `import`, but I decided that was out of scope for this PR.